backend/lib/github/webhookParser.js

```diff
@@ -0,0 +1,46 @@
+//
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+'use strict'
+
+const { createHmac, timingSafeEqual } = require('crypto')
+const bodyParser = require('body-parser')
```

Comment:
The body-parser middleware should not be inside this file. It should be in the file where the router is defined.